docs/accessControl.md

```diff
@@ -208,7 +208,9 @@ The `showClosedAssessmentScore` access rule restriction is often used in conjunc
 
 The `active` access rule restriction is useful for allowing students to see what assessments they have coming up. It should also be used when [returning exams to students](faq.md#how-do-i-give-students-access-to-view-their-exams-after-they-are-over). If `active` is set to `false`, students can see the assessment on the Assessments page, but they cannot start the assessment, create a new assessment instance, or submit answers to questions. If an assessment is currently not active but will be in the future, students can see when the assessment will become active by looking at the `Available credit` column on the Assessments page. The `active` property in an access rule is `true` by default.
 
-**Note**: if `active` is set to `false` in an access rule, the available `credit` cannot be set to any value other than 0 (the default value).
+If `active` is set to `false` in an access rule, the available `credit` cannot be set to any value other than 0 (the default value).
+
+Note that when `"active": false` is used to make assessments visible to students after they've finished the assessment, students can still access and modify files in any [workspaces](workspaces/index.md) associated with questions on the assessment.
```

Comment:
Maybe add a note that they can modify, but not submit?

Reply:
Done in fca0173, how does that look?